ecl/hthor/hthor.cpp

```diff
@@ -8391,6 +8391,11 @@ void CHThorBinaryDiskReadBase::append(IKeySegmentMonitor *segment)
     }
 }
 
+void CHThorBinaryDiskReadBase::append(FFoption option, IFieldFilter * filter)
```

Comment:
Just as a point of style, should we be using enum classes rather than prefixes like FF to distinguish our enum namespaces?